system-test/Dockerfile.node10-alpine

```diff
@@ -3,8 +3,13 @@ RUN apk update && apk upgrade && apk add --no-cache git
 WORKDIR /root/
 RUN go get github.com/google/pprof
 
+
 FROM node:10-alpine
-RUN apk update && apk upgrade && apk add --no-cache bash python g++ make
+
+ARG BUILD_BINARIES
+
+RUN apk update && apk upgrade && apk add --no-cache bash
+RUN if [ "$BUILD_BINARIES" = "true" ] ; then apk add --no-cache python g++ make ; fi
```

Comment:
For all Dockerfiles it seems it would be easier to have something like ADDITIONAL_PACKAGES as the build arg and pass it as --build-arg ADDITIONAL_PACKAGES="python g++ make" to the docker build commands from the wrapper scripts. No need for the if statement, no need to split the commands, should literally be one line change.